lib/containers/urls.pm

```diff
@@ -289,14 +289,29 @@ sub get_3rd_party_images {
     #     on z13: "Fatal glibc error: CPU lacks VXE support (z14 or later required)".
     # - ubi9 images require power9+ machine.
     #     on Power8: "Fatal glibc error: CPU lacks ISA 3.00 support (POWER9 or later required)"
-    # - poo#72124 Ubuntu image (occasionally) fails on s390x.
-    # - CentOS image not available on s390x.
+    # - ubi9 images require x86_64-v2, which needs certain cpu flags
+    my $can_run_ubi_9 = 1;
+    my $cpu_flags = script_output('lscpu | grep -i flags');
+    if (is_s390x || is_ppc64le) {
+        $can_run_ubi_9 = 0;
+    } elsif (is_x86_64) {
+        foreach my $flag (qw(cx16 lahf popcnt sse4_1 sse4_2 ssse3)) {
```

Comment:
This is probably something we will start to need in more tests, might be worthy to have a function checking for requirements of x86_64-v<n> level

Reply:
Done. Good idea :)